### PR TITLE
Refactored Trace to dynamically render its fields

### PR DESCRIPTION
**Pull request by lucasPDY**
Taiga: http://taiga.corplite.com/project/brian-qr-covid-19-tracker/us/98?kanban-status=129
**Description:** 
Previously, the fields in Trace.vue were hard-coded and static. 
With the implementation of `TracingForm.vue` (taking inspiration from `ProfileForm.vue`), `Trace` now sends an array of field data to `TracingForm` to render, enabling dynamic field rendering.

### CODE REVIEW

**Comment by khanguslee** [comment]
Let's wait for https://github.com/dtruong0/covid-tracer-aws/pull/28 to be pulled in first. You will need to update your local branch once that's in anyway.

File: src/components/TracingForm.vue [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+                :placeholder="field.placeholder"
+                :rules="field.rules"
+                :type="field.type"
+              />
+            </b-field>
+          </div>
+        </div>
+        <div id="formButtonContainer" class="buttons">
+          <b-button type="is-secondary" @click="create">Submit</b-button>
+          <!-- <b-button id="btnCheckInAnotherPerson" type="is-primary">Submit & Check-in Another Person</b-button> -->
```
Comment (khanguslee) [inline src/components/TracingForm.vue:21]: I know this was here before, but let's just remove this now. It's easy to put back anyway.

**Comment by khanguslee** [comment]
Can you resolve conflicts @lucasPDY

**Comment by lucasPDY** [comment]
Closing this PR due to multiple conflict resolution mistakes. A new PR has been opened at : https://github.com/dtruong0/covid-tracer-aws/pull/40

**Pull request closed without merging**